### PR TITLE
Aegis class long term care vessel

### PR DESCRIPTION
**Pull request by Draggeru**
## About The Pull Request

Adds the SSV Aegis, a roleplay-focused medical and service specialized ship to aid or take over rescue class ship cases that are above the critical care level that the rescue class level can handle, such as psychological treatment or long-term patient boarding.

## Why It's Good For The Game
Adds another syndicate-aligned faction medical/service ship(mostly roleplay oriented), this ship was designed with not only a mid/high crew in mind but with the added potential to accept and host other non-crew members for a length of time for whatever roleplay flavor. I really need to stress this ship was designed with a primary roleplay focus, it was designed in a way where people could spend weeks and even months on this one ship alone. This is what sets the Aegis apart from the Geneva, the Geneva is more gameplay oriented in comparison. The Aegis also features a joinable psychologist role and a joinable long-term patient role. Both of these roles are roleplay oriented.

aegisfinal4

aegisfinalingame

For lore: Your outstanding orders from the NSV are to respond to any major critical care situations where the patient may potentially need to be onboarded for any length of time, or in cases where first response ships have a major patient overflow. While you are not necessarily a first response vessel you are able to uphold such needs in the case of an emergency where rescue class vessels cannot respond.

The Aegis does NOT have any form of research potential, only housing a singular medical tech fab. If you somehow end up with research potential you should not spec into cargo/engineering/security. Your primary focus is to uphold NSV's orders and provide the care you specialize in.

## Changelog
:cl:
add: Adds the Aegis, a ship designed primarily for medical and service(and roleplay)
:cl:

### CODE REVIEW

**Comment by Yellow-Mushroom** [comment]
I feel as if this ship strongly overlaps with the syndicate Geneva class, with these both being syndicate medical ships with an included botany.

**Comment by Draggeru** [comment]
Geneva uses oil thrusters, contains a considerable amount of equipment for its size on top of being a major size difference, and is meant as a rescue vessel than an actual hospital in terms of roleplay flavor. Geneva also contains an emag.

**Comment by Draggeru** [comment]
This is officially done and ready for maintainers to pick at, I do not have any plans to alter it at this time

**Comment by Yellow-Mushroom** [comment]
Unsure if Im just misremembering how it looked, but with the new commits, could the PR get an updated pic?

**Comment by ZephyrTFA** [comment]
I've updated the PR body to contain the mapdiffbot rendered image

**Comment by Draggeru** [comment]
I have additional ideas for this PR and will be moving it to a draft after maptainers give the current version a look over. Current plans are to: reduce the chem lab to just boards that need to be built, replace the chem area with a psychologist office, shuffle the roles around to add psychologist and an "Onboard Patient" roleplay roles, and tweak the external access so it is in a more logical place. This ship is for the NSV and while it does overlap with the Geneva I think the roleplay focus, size, and the fact the NSV would have more ships of this type than most factions could justify the Aegis co exsisting with the Geneva

Edit:
Never mind, I wasn't patient enough to do this. New changes: Shifted the roles around and removed two deck assistant slots. Added one psychologist slot, one long-term patient slot, changed parts of the ship around that made the walls (specifically in cargo and in dorms) look better, and added a psychologist office, reduced all the chemistry equipment to just tech boards tucked away in medical storage.

aegisfinalforreal

File: _maps/configs/aegis.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	},
+	"Deck Service Assistant": {
+		"outfit": "/datum/outfit/job/botanist/syndicate/nsv",
+		"slots": 2
+	},
+	"Wrecker": {
+		"outfit": "/datum/outfit/job/miner/syndicate",
+		"slots": 1
+	},
+	"Ship Psycologist": {
```
Comment (Fikou) [inline _maps/configs/aegis.json]: misspelled

**Review by Latentish — CHANGES_REQUESTED** [review]
If I'm very honest with you, I'm not super impressed with this ship. Part of the reason I asked you to focus on ruin making first was so that you could also learn a lot of our more obscure requirements.
- Double walls are generally something you should avoid, which are all over this ship.
image
- The general shape of the ship isn't bad, it has a sort of star wars rebel alliance symbol going on. But the silhouette is rather chunky. I'd like to see you smooth out your ship's outline with more diagonal and smooth walls. As is you got this jpeg edge artifacting look to it, like you used the fill tool to make the shape but didn't trim it up.
image
- I like that you squeezed the botany into this little area, but botany on an NSV medical ship seems a bit out of place. I'm also just not a fan of the shape of this botany in general, as it's basically just a thin long rectangle.
image
- I like the medbay, I'd even suggest expanding it a bit.
image

**Comment by Draggeru** [comment]
New checklist before the next review:
[ ] Ensure there are no double walls
[ ] Restore the original shape of the ship, which is more symmetrical in the original first iteration
[ ] Scrunch? Botany down more so it is not as prominent but still useable for the kitchen
[ ] Rework medical to be larger

**Comment by Apogee-dev** [comment]
Don't forget to update the ship's image at the top of the PR.

**Comment by Draggeru** [comment]
This is officially done and ready for review, if I make any more changes that are not fixes yell at me

**Comment by Latentish** [comment]
I don't have the time for a real review right now but this is a massive improvement.

**Comment by Latentish** [comment]
image
This is very nice.

**Comment by SweetBlueSylveon** [comment]
Two things
image
I really like that you went with maintenance here, and I love the shape. My minor gripe is that this maintloot is sorta just, in the middle of these doorway areas. They'd serve better on racks or in a dumpter/clothing bin/chest. It just seems like an afterthought to put the equipment in. You could alternatively scatter it a bit, put gloves in one location, trashdrops in another, and gear in another.

**Comment by Latentish** [comment]
Last comment was me being stupid and blind. This is great. I love the maint aspect. Way to pack everything in.
image

**Comment by Latentish** [comment]
image
This feels empty, you should add stools in front of the bar, and add some clutter to this hallway.

**Review by Latentish — CHANGES_REQUESTED** [review]
Looking great so far! And fuck you you're getting a review anyway.
- Add a smartfridge to botany please. You could even put it in between the bar and botany, in the wall.
image
- The hallway infront of the bar seems really empty. You should add some stools in front of the bar counter, and put some clutter in the hallway. Tables, vending machines, etc.
- Optionally you could give them some more cooking equipment. An oven would be really appropriate for this setting, could put a little minikitchen area in that empty strip of floor in botany. But if you think it impedes the goal or balance of the ship that's more flourish than anything.
- I can't tell if you added one, but make sure there is a cargo/outpost communications console somewhere, with an area to receive packages.
- I can see you used the base path of "clutter" for your floors. You shouldn't use this, it's description is "this is clutter", and it's just some grey pixels. Its subtypes are what's meant to be used. "clutter" is somewhat out of place in general. (I know I said add clutter earlier but I meant like furniture just to be clear).
- Add a shower somewhere.

**Comment by Draggeru** [comment]
> Looking great so far! And fuck you you're getting a review anyway.
> 
> * Add a smartfridge to botany please. You could even put it in between the bar and botany, in the wall.
>   image
> * The hallway infront of the bar seems really empty. You should add some stools in front of the bar counter, and put some clutter in the hallway. Tables, vending machines, etc.
> * Optionally you could give them some more cooking equipment. An oven would be really appropriate for this setting, could put a little minikitchen area in that empty strip of floor in botany. But if you think it impedes the goal or balance of the ship that's more flourish than anything.
> * I can't tell if you added one, but make sure there is a cargo/outpost communications console somewhere, with an area to receive packages.
> * I can see you used the base path of "clutter" for your floors. You shouldn't use this, it's description is "this is clutter", and it's just some grey pixels. Its subtypes are what's meant to be used. "clutter" is somewhat out of place in general. (I know I said add clutter earlier but I meant like furniture just to be clear).
> * Add a shower somewhere.

-Consider it done
-Consider it done
-By oven do you mean microwave oven? There is already a microwave on the kitchen side of the service area. If not I'm not sure what you're referring to. Additionally I can add the boards for more machines, such as a few more hydroponic tray boards and cooking machine boards. I would like to leave the space to give players a bit of freedom to alter it themselves. I think adding the boards would let them do that better anyways.
-Cargo console is on the bridge itself, I'll make sure the area is set and not broken
-Will fix
-There is already a shower in medical, you cannot see it but there is a directional window separating the bed and the shower there. Are you wanting another shower?

**Comment by Draggeru** [comment]
aegisfinal2electricbogaloo

**Comment by Latentish** [comment]
> -Consider it done -Consider it done -By oven do you mean microwave oven? There is already a microwave on the kitchen side of the service area. If not I'm not sure what you're referring to. Additionally I can add the boards for more machines, such as a few more hydroponic tray boards and cooking machine boards. I would like to leave the space to give players a bit of freedom to alter it themselves. I think adding the boards would let them do that better anyways. -Cargo console is on the bridge itself, I'll make sure the area is set and not broken -Will fix -There is already a shower in medical, you cannot see it but there is a directional window separating the bed and the shower there. Are you wanting another shower?

- I meant the normal convection oven. But citcuits work fine for this.
-  When it comes to a shower, yeah a shower in medical is always a good addition. But from an rp and living perpective it would be kinda whack that a patient would need to go to medical just to take a shower. Try to put it near the normal bothrooms, or in there outright if you can manage.

**Comment by Latentish** [comment]
> aegisfinal2electricbogaloo

Good.

**Review by Latentish — DISMISSED** [review]

**Review by Latentish — APPROVED** [review]
Seems pretty sanitary so far. I'd like to see it get tested. TM?

**Comment by Draggeru** [comment]
After discussing issues with my repository with maintainers this map pr is moved to #1391 on a new repository.

**Pull request closed without merging**